packages/foundation/src/secrets/app_secrets.skeleton.ts

```diff
@@ -28,3 +28,7 @@ export class AppSecrets {
   }
 }
```

Comment:
Please dont uncomment this. Make a copy of this with app_secrets.ts

Reply:
change the file and make copy of app_scrects.ts and push